src/viking.js

```diff
@@ -1,14 +1,87 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor (health, strength){
+    this.health = health
+    this.strength = strength
+  }
+  attack(){
+return this.strength
+  }
+  receiveDamage(damage){
+    this.health -= damage
+
+  }
+
+}
 
 // Viking
-class Viking {}
+class Viking extends Soldier {
+constructor (name, health, strength){
+super(health, strength);
+this.name = name
+}
+attack(){
+  return this.strength
+
+} 
+receiveDamage(damage) {
+  this.health -= damage
+  if (this.health > 0) {
+  return `${this.name} has received ${damage} points of damage`
+  } else {
+  return `${this.name} has died in act of combat`  
+  }
+  
+}
+battleCry(){
+  return 'Odin Owns You All!'
+  }
+
+
+}
 
 // Saxon
-class Saxon {}
+class Saxon extends Soldier {
+ 
```

Comment:
Aun que saxon herede de soldier las propiedades tenemos que indicarse lo en su constructor y hacer uso de super para accerder a las keys y metodos del padre
```suggestion
 constructor ( health, strength){
super(health, strength);
}
```